subjects/blockchain/node-dashboard/audit/README.md

```diff
@@ -0,0 +1,37 @@
+## Node Dashboard audit
+
+##### Read the documentation
+###### Does the README file contains the instructions to launch the project?
+###### Are two different clients launched?
+###### Is clique the consensus algorithm?
+
+##### Launch the network with the `launch-network` and display its status with `infos`
+###### Does the command launch the five nodes?
+###### Does the interface display the latest block of each node?
+###### Does the interface display their Ethereum address and balance?
+###### Does the interface display the CPU and memory consumption of each node?
+
+##### Launch scenario 0
+###### Does Alice, Bob and Cassandra have a positive balance of ETH?
+
+##### Launch scenario 1
+###### Does the command provide feedback?
+###### Does the `infos` command show the updated ETH balance? 
```

Comment:
the code rendering for the `h6` in the audit is a bit off: it always renders it as a code block (new line with just this word). I would use bold or italics to highlight the keyword instead